src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenConstructorInitTests.cs

```diff
@@ -1198,7 +1198,7 @@ class C
                 instance void .ctor () cil managed
         {
                 // Method begins at RVA 0x207f
-                // Code size 7 (0x7)
+                // Code size: 7 (0x7)
```

Comment:
new C# decompiler syntax.

Reply:
Sorry for the change... I changed this for consistency with other info we print and did not realize in time that ildasm does not use a colon...